Automattic/jetpack
# VideoPress: Enforce chapters restrictions on description parsing

Fixes #28588

## Proposed changes:

* Includes the chapters validation to the check performed before generating the chapters file
* Adds a notice in the Details Panel when there is an incomplete chapters list

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add a VideoPress block
* Add incomplete chapters and check that a message is shown
* Save the post
* Check that the chapters were not added to the video

An incomplete chapter list has at least one of the following:
- less than 3 chapters
- less than 10 seconds in between any two chapters
- first chapter with a start time other than `00:00` or `00:00:00`
- any chapter without a title

![2023-02-06_11-07-22](https://user-images.githubusercontent.com/8486249/216993947-e945c676-4420-4887-a1c7-e5102e3ea7a8.png)
